README.md

```diff
@@ -14,6 +14,9 @@ minor version bump.
 `halo2` currently uses [rayon](https://github.com/rayon-rs/rayon) for parallel computation.
 The `RAYON_NUM_THREADS` environment variable can be used to set the number of threads.
 
+You can disable [rayon] by disabling the ["multicore"] feature.
+Warning! Halo2 will lose access to parallelism if you disable the ["multicore"] feature. This will significantly downgrade performance. 
```

Comment:
```suggestion
Warning! Halo2 will lose access to parallelism if you disable the ["multicore"] feature.
This will significantly degrade performance.
```